test/test_sequence_transformer.py

```diff
@@ -166,7 +166,7 @@ def test_tsfresh_all_nan():
     tsfresh_features_median = np.median(computed_tsfresh_features, axis=0)
     imputed_tsfresh_features_expected = np.repeat(tsfresh_features_median.reshape((1, -1)), 2, axis=0)
     assert_array_almost_equal(X_observed, X_expected)
-    assert X_with_features.shape == (4, 791)
+    assert X_with_features.shape == (4, 13)
```

Comment:
please add a comment explaining the number 13 (if tsfresh update their lengths, it will make life easier for the maintainers of this package.
This comment applies to other places with numbers derived from the choices of tsfresh (e.g. the 10 in `_compute_expansion_threshold`)

Reply:
added.